utils/watchClass.util.ts

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line
+export function watchClass(targetNode: any, classToWatch: any){
+  let lastClassState = targetNode.classList.contains(classToWatch);
+  const observer = new MutationObserver(mutationsList => {
+    for (let i = 0; i < mutationsList.length; i++) {
+      const mutation = mutationsList[i];
+      if (
+        mutation.type === "attributes" &&
+        mutation.attributeName === "class"
+      ) {
+        // @ts-ignore
```

Comment:
precisa desses ts-ignore agora?

Reply:
Sim, caso contrário da esse erro: `Property 'classList' does not exist on type 'Node'.ts(2339)`